src/docs/product/dashboards/custom-dashboards/index.mdx

```diff
@@ -89,4 +89,15 @@ transactions'.
 
 ## Data Set Selection
 
-Select between events and [issues](/product/issues/) data in a widget with the "Data Set" selector. Choosing "Issues" allows you to query using issues filters, such as `is:unresolved` for unresolved issues, and to display and sort by issues fields. This feature only applies to [table](#table-results) visualizations.
+Select between events and [issues](/product/issues/) data in a widget with the "Data Set" selector.
```

Comment:
Should we link to some events docs like: https://docs.sentry.io/product/accounts/pricing/#events ?
Kind of weird to link to the pricing page. Isabel has an entry for `event` under key terms, maybe we just sync up later and fix the link

Reply:
Hmm, took a look around at what existing events doc we have. I think the closest thing is this page? https://docs.sentry.io/product/sentry-basics/enrich-data/
But it contains quite a lot of info outside of just events so I'm not sure if it would be confusing to users.

The Key Terms pr got merged here https://docs.sentry.io/product/accounts/quotas/#key-terms which we could link to as well. 

I think the first link might be more suitable since the Key Terms page is more around quota management or we could leave it unlinked for now. I don't have a strong opinion on this so maybe @jas-kas can make the call?

Reply:
I don't think it's necessary to link an Events doc, since we already link to Discover documentation and that would be closest representation of what we're referring to. Additionally, we're also changing this name from "Events" to "Errors and Transactions" in the near future as well.

Reply:
There is a definition on this page:
https://docs.sentry.io/product/sentry-basics/key-terms/

However, you can't link directly to the definition, so I'd agree with Jasmin to not worry about linking out at this point.